ZichanYang324/BF-Employee-HR
# BGP-30 -[frontend and backend] - Login page

# Description
Add login page with front end and backend, also add UI from react material UI.

## Related PRS (if any):
please checkout to this PR #19  and test the register function, then start testing on this PR

## Main changes explained:
- Add redux for login employee client
- Add dashboard and Nav bar
- Add cookies
- Add token


## How to test:
1. check into current branch
2. do `cd Server` and separate another terminal do `cd Employee-Client`
3. do `npm install`  and `npm run dev` in both frontend and backend to run this PR locally
4. add routes `.../login`
5. verify function “login” with username and password

## Screenshots or videos of changes or expecting result:
![Screenshot 2024-02-25 at 13 01 50](https://github.com/ZichanYang324/BF-Employee-HR/assets/86720318/becd9954-297f-48bf-9583-53a3b9bf9711)
![Screenshot 2024-02-25 at 13 01 57](https://github.com/ZichanYang324/BF-Employee-HR/assets/86720318/91cde4ff-f249-4878-9692-42ba7f991b8f)




## Note:
If you want to log in and then logout try another login, you will need to clear the cookies, sessions, and local storage(you can do this in the inspect -> application -> storage(clear these part, including cookies, sessions, and local storage)
